packages/forma-36-website/src/components/Container.js

```diff
@@ -49,34 +49,34 @@ const styles = {
 };
 
 const markToComponentMap = {
-  h1: props => <DisplayText {...props} />,
-  h2: props => <Heading element="h2" {...props} />,
-  h3: props => <Subheading element="h3" {...props} />,
-  h4: props => <Subheading element="h4" {...props} />,
-  h5: props => <Subheading element="h5" {...props} />,
-  h6: props => <Subheading element="h6" {...props} />,
-  p: props => <Paragraph className="f36-font-size--l" {...props} />,
-  a: props => <TextLink {...props} />,
-  ul: props => <List className="f36-margin-bottom--m" {...props} />,
-  li: props => (
+  h1: (props) => <DisplayText {...props} />,
+  h2: (props) => <Heading element="h2" {...props} />,
+  h3: (props) => <Subheading element="h3" {...props} />,
+  h4: (props) => <Subheading element="h4" {...props} />,
+  h5: (props) => <Subheading element="h5" {...props} />,
+  h6: (props) => <Subheading element="h6" {...props} />,
+  p: (props) => <Paragraph className="f36-font-size--l" {...props} />,
+  a: (props) => <TextLink className="f36-font-size--l" {...props} />,
```

Comment:
Link the same size as Paragraph, but I think we need to do something global maybe if we really want to overwrite font-size, to be sure that it is consistent. WDYT?

Reply:
Yeah it's a bit messy but I think this is all that's needed.

We want these markdown elements to use a larger base font size for readability, but we want the actual components in the code playgrounds to use our default F36 base font size. 

Because we're using REMs we can't target one part unless I've forgotten how to CSS, so this seems like the best approach